3rdparty/CMakeLists.txt

```diff
@@ -6,4 +6,4 @@ add_subdirectory(tmatrix)
 add_subdirectory(wigner)
 add_subdirectory(Faddeeva)
 add_subdirectory(gui)
-add_subdirectory(pybind11)
+#add_subdirectory(pybind11)
```

Comment:
just remove?

Reply:
Good catch, that is was not supposed to be left there...